.github/workflows/build-test-nix.jsonnet

```diff
@@ -0,0 +1,57 @@
+// This workflow builds and tests semgrep via nix!
+local actions = import 'libs/actions.libsonnet';
+local gha = import 'libs/gha.libsonnet';
+
+// ----------------------------------------------------------------------------
+// The Workflow
+// ----------------------------------------------------------------------------
+local job = gha.os_matrix(
+  oss=['ubuntu-latest', 'macos-latest'],
+  steps=[
+    gha.speedy_checkout_step,
+    actions.checkout_with_submodules(),
+    {
+        name: "Set up Nix",
+        uses: "DeterminateSystems/nix-installer-action@main",
+        with: {
+            "extra-conf": "sandbox = false"
```

Comment:
would be good to comment why we need this sandbox = false. What happens if we put true? What is failing?

Reply:
will add a comment